src/alchemlyb/estimators/ti_.py

```diff
@@ -105,3 +105,45 @@ def fit(self, dHdl):
         self.states_ = means.index.values.tolist()
 
         return self
+
+    def separate_dhdl(self):
+        """
+        For transitions with multiple lambda, this function will separate the
+        dhdl with multiple columns into a list of Dataframe with a single column
```

Comment:
Explain more clearly what the list contains. Is this a list of multiple DataFrame instances where each DataFrame contains ... what?

Explain it to someone who does not have the data structure in front of themselves.

Mention why this might be useful.